packages/xo-server/src/xapi/index.mjs

```diff
@@ -1382,6 +1383,7 @@ export default class Xapi extends XapiBase {
     return find(this.objects.all, obj => obj.$type === 'SR' && canSrHaveNewVdiOfSize(obj, minSize))
   }
 
+  @decorateWith(debounceWithKey, 60e3, hostRef => hostRef)
```

Comment:
IMHO, every hour is already a very very enough for this use case, what do you think?

Reply:
I think 10 minutes is already enough to not scale by the number of opened xo-web, which is already a big win :slightly_smiling_face: 

We don't want the warning to keep showing for a long time even after the user fixed NTP on its host.

Reply:
@olivierlambert If it's okay for, please resolve this comment :slightly_smiling_face:

Reply:
Fine by me then :)